frontend/calangute-beach/style.css

```
@@ -0,0 +1,587 @@
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: background 0.2s;
+    z-index: 10;
+}
+
+.lightbox-close:hover { background: var(--primary-color); }
+
+/* Animation Keyframes */
+@keyframes fadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)